bertopic/representation/_cohere.py

```diff
@@ -33,7 +33,9 @@
 Sample texts from this topic:
 [DOCUMENTS]
 Keywords: [KEYWORDS]
-Topic name:"""
+Provide the topic name directly without any explanation."""
```

Comment:
Why did you make this change? In my experience, by providing the model with a pre-fix, there is no need for mentioning that it should provide the topic name without any explanation.

Reply:
Based on my experiments, the pre-fix prompt works well for completion models but not with (small) chat models. You can try different prompts locally to see the difference and feel free to change/iterate the prompt.